src/answers-headless.ts

```diff
@@ -162,13 +173,18 @@ export default class AnswersHeadless {
     this.stateManager.dispatchEvent('searchStatus/setIsLoading', false);
     this.stateManager.dispatchEvent('meta/setUUID', response.uuid);
     this.stateManager.dispatchEvent('directAnswer/setResult', response.directAnswer);
+    if (this.state.query.searchAggregation?.enabled) {
+      this.stateManager.dispatchEvent('query/setSearchAggregationId', uuidv4());
```

Comment:
could you expand a little on how searches will be grouped together? It looks like here we would generate a new autocompleteSessionId for every single universal search made?

Reply:
After discussing with product, the usage of autocompleteSessionId is different from the functionality added in this pr. autocompleteSessionId should be used in vertical/universal search only when it's executed for the purpose of visual autocomplete (the search term aggregation and this id doesn't apply for normal autocomplete). My original thought was to update uuid after every final search is triggered, and the visual autocomplete would have another function to execute autocomplete + search endpoints. Updated to always include autocompleteSessionId, and user can set that if they want a uuid to be included in the request